MonitoringSystem/Client/MonitoringReporter.py

```diff
@@ -0,0 +1,133 @@
+"""
+This class is used to insert data to elasticsearch. It uses an internal list which is used to keep messages in the memory. 
```

Comment:
Well, this class does not HAVE to insert data to an ElasticSearch instance, IIUC it inserts everywhere when a `put` function is provided